doc/radosgw/config-ref.rst

```diff
@@ -416,6 +382,73 @@ instances or all radosgw-admin commands can be put into the ``[global]`` or the
 :Default: ``true``
 
 
+Garbage Collection Settings
+===========================
+
+The Ceph Object Gateway allocates storage for new and overwritten objects
+immediately. Additionally, the parts of a multi-part upload also consume
+storage.
```

Comment:
this  para can be  completely dropped, or at least the second sentence, as it just simply stats the parts consume storage